wicket-native-websocket/wicket-native-websocket-core/src/main/java/org/apache/wicket/protocol/ws/api/WebSocketResponse.java

```diff
@@ -87,13 +100,34 @@ public void close()
 			{
 				if (text != null)
 				{
-					connection.sendMessage(text.toString());
+					if (asynchronous)
+					{
+						if (timeout > 0)
```

Comment:
I would drop this `if` in favor of `connection.sendMessageAsync(text.toString(), timeout);`

Reply:
Done